src/tests/fake/user.tsx

```diff
@@ -101,3 +103,29 @@ export function mockApiFindUsers(users: Array<User>, query?: Query) {
             items: users,
         });
 }
+
+/**
+ * Mocks an API call for getting the user details
+ *
+ * @param userId - The users unique id
+ * @param userDetail - The details of the user
+ * @returns A nock scope for the mocked API call
+ */
+export function mockApiGetUser(userId: string, userDetail: User) {
```

Comment:
```suggestion
export function mockApiGetUser(userId: string, user: User) {
```
The way we have named our types `userDetailed` is the default state whereas the reduced cases are the special cases